src/js/collect/NavBar.jsx

```diff
@@ -59,15 +60,20 @@ export default function NavBar({ shiftPlotId, goToPlot, nextPlot, prevPlot, setP
         }}
         value={currentPlotNumber + ""}
         onClickIncrease={() => {
-          setCurrentPlotNumber(currentPlotNumber + 1);
+          const newNumber = currentPlotNumber + 1;
+          if (newNumber <= maxPlotNumber) setCurrentPlotNumber(newNumber);
         }}
         onClickDecrease={() => {
-          setCurrentPlotNumber(currentPlotNumber - 1);
+          const newNumber = currentPlotNumber - 1;
+          if (newNumber > 0) setCurrentPlotNumber(currentPlotNumber - 1);
         }}
         extraStyle={{ marginRight: "0.5rem" }}
       />
       <Button
         onClick={() => {
+          if (currentPlotNumber < 1 || currentPlotNumber > maxPlotNumber) {
+            alert(t("validate.invalidPlot"));
+          }
```

Comment:
This is a nice feature add. Can you change this to use the messageBox/modal functionality instead of just an alert?